src/main/java/uk/gov/digital/ho/pttg/testrunner/SmokeTestsService.java

```diff
@@ -19,6 +19,7 @@
 @AllArgsConstructor
 public class SmokeTestsService {
 
+    private static final String TEST_NINO = "AA000000A";
```

Comment:
```suggestion
    private static final String TEST_NINO = "QQ123456C";
```
Might want to change this to be an actually fictional NINO

Reply:
@zuzak Is QQ123456C the accepted "fake"?

Reply:
@zuzak That NINO doesn't appear to be valid. IPS checks that the NINO is valid before proceeding. Going to have to update IPS unless there's a "fictional" but valid NINO

Reply:
> @zuzak Is QQ123456C the accepted "fake"?

It's the usual one to use as an example in UIs, but I guess it's deliberately invalid.
https://design-system.service.gov.uk/patterns/national-insurance-numbers/